SETUP.md

```diff
@@ -0,0 +1,189 @@
+# Setting up your Cook dev environment
```

Comment:
I vaguely remember a comment saying you had reordered this doc so that the mac specific stuff was separated or noted clearly. Is there a missing commit?